### PR TITLE
Chore/users migrations

### PR DESCRIPTION
**Pull request by JohanHjelsethStorstad**
DOBBEL OMEGA
This PR adds user migrations.

### CODE REVIEW

**Comment by Paulijuz** [comment]
Took a quick look now and it looks good. I'll do a proper review once #328 is merged and the conflicts are fixed.

**Comment by JohanHjelsethStorstad** [comment]
@Paulijuz. - now rebased on main

File: src/prisma/prismaservice/src/dobbelOmega/migrateUsers.ts [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+                bio: user.bio ?? '',
+                acceptedTerms: undefined,
+                sex: sexMap[user.sex],
+                allergies: undefined,
+                mobile: undefined,
+                emailVerified: undefined,
+                createdAt: user.createdAt,
+                updatedAt: user.updatedAt,
+                imageId: vevenIdToPnId(imageIdMap, user.ImageId),
+                archived: user.archived,
```
Comment (theodorklauritzen) [inline src/prisma/prismaservice/src/dobbelOmega/migrateUsers.ts:92]: I don't think it is necessary to archive PN users.

**Review by theodorklauritzen — APPROVED** [review]
Great work!

It's probably nice to store if users are archived. However I don't think it's necessary to archive users. I think they still should be able to login, but with limited permissions

**Pull request merged**